.github/workflows/release.yml

```diff
@@ -0,0 +1,308 @@
+name: Build + draft release
+
+# (erodkin) this is needed to create the release, per https://github.com/softprops/action-gh-release/issues/236
+permissions:
+  contents: write
+
+jobs:
+  prepare:
+    if: github.repository_owner == 'viamrobotics'
+    runs-on: buildjet-8vcpu-ubuntu-2204-arm
```

Comment:
This can probably happen on just `ubuntu-latest`. Since there's no actual build step, we don't need a large buildjet instance